### PR TITLE
Allow skipping preflight tests using PR labels

### PR DESCRIPTION
**Pull request by Allda**
By adding a label to a PR in format tests/skip/{test_name} admins can skip certain tests and make it optional.

JIRA: ISV-4316

### CODE REVIEW

**Review by mantomas — APPROVED** [review]
LGTM

File: ansible/roles/operator-pipeline/templates/openshift/tasks/evaluate-preflight-result.yml [hunk 1]
```diff
@@ -53,7 +53,12 @@ spec:
       workingDir: $(workspaces.input.path)
       image: "$(params.pipeline_image)"
       env:
-        - name: GITHUB_TOKEN
+        - name: GITHUB_TOKEN # Token used by internal CLI tool
+          valueFrom:
+            secretKeyRef:
+              name: $(params.github_token_secret_name)
+              key: $(params.github_token_secret_key)
+        - name: GH_TOKEN # Token used by the GH tool
```
Comment (mporrato) [inline ansible/roles/operator-pipeline/templates/openshift/tasks/evaluate-preflight-result.yml:61]: Is this really needed? According to the [docs](https://cli.github.com/manual/gh_help_environment), the `gh` tool should work with either `GITHUB_TOKEN` or `GH_TOKEN`.

**Review by mporrato — APPROVED** [review]
LGTM, but this won't have the same behaviour as the current pipeline because it will require manual interaction.
We should probably add a new task at the beginning of the pipeline to automatically apply some skip-labels based on the pr contents (for example extract the allow-list from the existing `ci/pipeline-config.yaml` or perhaps adding something similar to `config.yaml` and check if the affected operator matches any of the names), but this can be done in another PR.

File: ansible/roles/operator-pipeline/templates/openshift/tasks/evaluate-preflight-result.yml (hunk 1 above)
Comment (Allda) [reply ansible/roles/operator-pipeline/templates/openshift/tasks/evaluate-preflight-result.yml]: Ah, good point. I removed the this section.

**Pull request merged**